### ISSUE TITLE
UC AMANDA Automated Email Clutter (NN)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#2952]
### As a UC folder applicant, I want system generated emails to efficiently provide me with the information I need to know. 

Due to system programming on the process tab, the UC folder sends out 3 almost identical emails to the folder applicant, usually in rapid succession. 

**Stakeholders:** Jacqueline Lozano, isaiah lewallen
**DTS Manager:** Nate

See issue comments for examples of each email that is sent, and the corresponding process that sent the email. 

**Processes that send the emails:**
Application Review (82005), Permit Completeness check (51065), Web Application Acceptance (70000) and Initial Distribution (51070)

**Acceptance Criteria**

- Only one email should be sent to the Applicant, instead of the 2 or 3 being sent (Process to send Email: TBD on Monday 8/3/2020)

- Set one process to be the Email Trigger. Right now they are Web Application Accepted and Permit Completeness Check (Trigger: TBD on Monday 8/3/2020)

- Disable/remove email trigger from other Process

- Update Verbiage on emails (on #3536)

1. Remove Verbiage in all UC emails stating:
"Deliver one (1) half-size paper plan set to Austin Water at 625 E 10th St (Attention: AW-Infrastructure Management AULCC Review) on the Wednesday two weeks prior to your scheduled AULCC meeting and include a screen print displaying your case, tracking or folder number. Bring a plan set to your scheduled AULCC meeting. "  
2. Verbiage to be changed in the Permit Completeness Check email:
**From** "10th Floor Conference Room, One Texas Center, 505 Barton Springs Road"
**To** "Online via Microsoft TEAMS. You will receive an email invitation with the TEAMS link and optional call in number at a later date"
3. Meeting Date and time will remain the same since they never change (Thursdays at 2pm)
4. Choose 1 of these processes to send one email, turn the email feature on the other 2 processes off. 

**Next Steps:**

- Find lookup table with emails
- Fix the verbiage on emails #3536
- Disable current triggers that send the email to users; Possible triggers Web Application Acceptance (70000) and Permit Completeness Check 51065
- Tie in the sending of email to the Initial Distribution (51070) and the Adjust Review Due Dates (61060)
- Make the Initial Distribution (51070) and the Adjust Review Due Dates (61060) processes not assign users automatically

**Comment by TracyLinder** [comment]
**Web Application Acceptance, Approved Attempt:**

Thank you for your submittal to the Austin Utility Location and Coordination Committee (AULCC).  Your Utility Coordination Application is accepted.

Please review your information. If any changes need to be made, please contact:  
isaiah.lewallen@austintexas.gov (512) 974-1479 
aulcc@austintexas.gov                   ROWID: 12388595
Project Name: 505 BARTON SPRINGS ROAD Unit  10
Applicant:  Tracy Linder
Project Contact: Not Provided
Project Description: TEST FEES FOR School  We've received all information necessary to distribute your case for review. Deliver one (1) half-size paper plan set to Austin Water at 625 E 10th St (Attention: AW-Infrastructure Management AULCC Review) on the Wednesday two weeks prior to your scheduled AULCC meeting and include a screen print displaying your case, tracking or folder number. Bring a plan set to your scheduled AULCC meeting.   For information about  AULCC please visit https://gcc01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.austintexas.gov%2FAULCC&amp;data=02%7C01%7CTracy.Linder%40austintexas.gov%7Cd270ef39026d436dfbd508d80cb39ee8%7C5c5e19f6a6ab4b45b1d0be4608a9a67f%7C0%7C0%7C637273312910635221&amp;sdata=iu%2F4w5nyPhZ8vQou8JVlz%2BYNjPP48%2FxqaSyagIbyOa8%3D&amp;reserved=0

**Comment by TracyLinder** [comment]
**Application Review, Complete Attempt**
Thank you for your submittal to the Austin Utility Location and Coordination Committee (AULCC).  Your Utility Coordination Application is accepted.

Please review your information. If any changes need to be made, please contact:  
isaiah.lewallen@austintexas.gov (512) 974-1479 
aulcc@austintexas.gov                   ROWID: 12388595
Project Name: 505 BARTON SPRINGS ROAD Unit  10
Applicant:  Tracy Linder
Project Contact: Not Provided
Project Description: TEST FEES FOR School  We've received all information necessary to distribute your case for review. Deliver one (1) half-size paper plan set to Austin Water at 625 E 10th St (Attention: AW-Infrastructure Management AULCC Review) on the Wednesday two weeks prior to your scheduled AULCC meeting and include a screen print displaying your case, tracking or folder number. Bring a plan set to your scheduled AULCC meeting.   For information about  AULCC please visit https://gcc01.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.austintexas.gov%2FAULCC&amp;data=02%7C01%7CTracy.Linder%40austintexas.gov%7Cee628c57d2254210fc8b08d80cb3d602%7C5c5e19f6a6ab4b45b1d0be4608a9a67f%7C0%7C0%7C637273313825038124&amp;sdata=hU9Civ9Umxkw4%2FBU%2BsSPlEGxzFKF3KzCL0vJWUbEYVM%3D&amp;reserved=0

**Comment by TracyLinder** [comment]
**Permit Completeness Check, Various Attempts:**
Hello: 
Thank you for your submittal to the Austin Utility Location and Coordinantion Committee (AULCC). Your ROWMAN Utility Coordinantion Application has been scheduled.

AULCC Meeting date: 18-JUN-20
Meeting Location: **10th Floor Conference Room, One Texas Center, 505 Barton Springs Road**     
Meeting Starts: 2pm 

Please review your information.  
If any data needs to be revised, added, or deleted; Please contact: 
isaiah.lewallen@austintexas.gov 
(512) 974- 1479

AULCC Tracking Number #:123456 ROWID: 12388595 Project Name:505 BARTON SPRINGS ROAD Unit  10 Applicant:  Tracy Linder
 Project Contact: No Organization Name  Project Description: TEST FEES FOR School  1.  Once the Project is reviewed, please  complete your Minutes worksheet to record responses, identify areas of conflict, and note proposed resolutions. 
-   Projects that have been through AULCC previously, must include all utility comments from previous reviews on the Minutes worksheet.		
-  Final review and clearance is  contingent on final review of the conflict/resolution data and concurrence of  the Project Engineer, Project Manager, Project Owner, and the AULCC Chairman.			
2.   A  AULCC Completeness  Letter will be granted once all the conflicts are  identified and resolution  verified. This letter is  required for General Permit, Site Plan, Site Plan Correction, or Site Plan Exemption submittal to Development Services Department Intake office on the 4th floor of One Texas Center.

**Comment by TracyLinder** [comment]
Kaarthy to begin scoping during sprint 29.

**Comment by TracyLinder** [comment]
@Nadin-Nader I just added you as DTS Manager

**Comment by Nadin-Nader** [comment]
Attachments from Tracy's email to Jaclyn's email:

[https://app.zenhub.com/files/140626918/74eb7b34-2218-4cc9-b4a2-03ad001631ad/download](https://app.zenhub.com/files/140626918/74eb7b34-2218-4cc9-b4a2-03ad001631ad/download)[https://app.zenhub.com/files/140626918/58b9b2e7-f699-44e2-ba9d-70edfda587ab/download](https://app.zenhub.com/files/140626918/58b9b2e7-f699-44e2-ba9d-70edfda587ab/download)

**Comment by Nadin-Nader** [comment]
**Status:** Meeting scheduled for 7/31/2020 at 11am with Jaclyn, Kaarthy and Robert

**Comment by Nadin-Nader** [comment]
@TracyLinder @licarir Updated the Acceptance Criteria and moving to On Deck. Next Meeting for this will be on Monday August 3, 2020

**Comment by Nadin-Nader** [comment]
In today's meeting we gathered the initial acceptance criteria for this issue to be resolved. The Gist of the issue is that the users are getting 2 to 3 emails when their application is accepted and/or reviewed. The users would like for the applicants to only receive one email for the UC folder. 

The information on the emails will also have to be changed (see acceptance criteria) and Card #3536

**Comment by Nadin-Nader** [comment]
In today's meeting with Isaiah , Jaclyn, Robert Kaarthy and Tracy. we were able to go over which process should be sending the email. 
The conclusion was to have the email tied to the Initial distribution and the Adjust Review Due Dates Process as they are used on all folders. 
We also discussed "turning off" the processes (via the code) that currently send the email from and switch it to just the two processes needed

**Next Steps:**

- Find lookup table with emails
- Fix the verbiage on emails #3536
- Disable current triggers that send the email to users; Possible triggers Web Application Acceptance (70000) and Permit Completeness Check 51065
- Tie in the sending of email to the Initial Distribution (51070) and the Adjust Review Due Dates (61060) 

@licarir Please let me know if i am missing anything from here if not pull in when ready after #3536

**Comment by TracyLinder** [comment]
**Robert to begin looking at this during sprint 31.**

**Comment by TracyLinder** [comment]
We'll create a new ticket if email updates are needed.
